index.html

```diff
@@ -484,14 +484,14 @@ <h3>Media Feed Item</h2>
                 numberOfSeasons</a> property MUST be a positive integer.
               </li>
               <li>
-                The <a data-cite="schema-org/episode#">episode</a> property MAY
```

Comment:
It's not letting me comment on the exact line, but above where it says numberOfEpisodes must be a positive integer, it may be good to note that numberOfEpisodes can be embedded in the TV season instead (according to the example).

Reply:
I removed this from TV series because I don't think we actually need it. If the TV series has multiple seasons then we only need the number of episodes on the season to know when we reached the end of the season. If the TV series has no seasons then we don't need the number of episodes because we don't need to know that we have run out of episodes